test/command/5876-datadir-metadata.md

```diff
@@ -0,0 +1,18 @@
+```
```

Comment:
Usually we just name these with the number, e.g. 5876.md.

Reply:
I'll try to write the other tests that you proposed too. There appears to be a convention for enumerating tests that belong to the same issue, e.g. `5876a.md`, `5876b.md`. Is that OK?

Should mention that [CONTRIBUTING.md](https://github.com/jgm/pandoc/blob/master/CONTRIBUTING.md#tests) suggests using both issue number and feature in the test file names.

Reply:
Changed in fixup.

Reply:
Oh, didn't realize that about contributing.md!
I guess it's probably not bad advice, and it looks like we've done it sometimes.
So, as you like.

For multiple tests belonging to the same issue, simply include them all in the same file.  A command test file can contain any number of command tests (separate code blocks).  We don't really have a convention of putting them in separate files with letter suffixes.

Reply:
Ah, I see. I gathered all tests in `5876.md` then. I think an extra suffix is unnecessary in this case since the tests are pretty clear about what is being tested.